src/RadixDlt.NetworkGateway.Abstractions/Model/PendingTransactionPayloadLedgerStatus.cs

```diff
@@ -0,0 +1,162 @@
+ * interaction with the Work;
+ *
+ * B. any failure in a transmission or loss of any token or assets keys or other digital
+ * artefacts due to errors in transmission;
+ *
+ * C. bugs, hacks, logic errors or faults in the Work or any communication;
+ *
+ * D. system software or apparatus including but not limited to losses caused by errors
+ * in holding or transmitting tokens by any third-party;
+ *
+ * E. breaches or failure of security including hacker attacks, loss or disclosure of
+ * password, loss of private key, unauthorised use or misuse of such passwords or keys;
+ *
+ * F. any losses including loss of anticipated savings or other benefits resulting from
+ * use of the Work or any changes to the Work (however implemented).
+ *
+ * You are solely responsible for; testing, validating and evaluation of all operation
+ * logic, functionality, security and appropriateness of using the Work for any commercial
+ * or non-commercial purpose and for any reproduction or redistribution by You of the
+ * Work. You assume all risks associated with Your use of the Work and the exercise of
+ * permissions under this License.
+ */
+
+using System;
+
+namespace RadixDlt.NetworkGateway.Abstractions.Model;
+
+/// <summary>
+/// This follows the convention from the Core API.
+/// </summary>
+public enum PendingTransactionPayloadLedgerStatus
+{
+    /// <summary>
+    /// We know that this specific payload has been committed as a success.
+    /// </summary>
+    CommittedSuccess,
+
+    /// <summary>
+    /// We know that this specific payload has been committed as a failure.
+    /// </summary>
+    CommittedFailure,
+
+    /// <summary>
+    /// We know that the payload has been committed, but don't yet know the outcome until it's synced (Success/Failure).
+    /// </summary>
+    CommitPendingOutcomeUnknown,
+
+    /// <summary>
+    /// We know that the intent has been committed with a different payload, but don't yet know the outcome
+    /// until it's synced (Success/Failure).
+    /// </summary>
+    CommitOfOtherPayloadForIntentPendingOutcomeUnknown,
+
+    /// <summary>
+    /// We know that this specific payload is permanently rejected.
+    ///
+    /// Note - this does not imply that the intent inside the payload is permanently rejected.
+    /// For example, if there are multiple payloads containing the same intent, only one payload
+    /// gets committed, and the rest get permanently rejected.
+    /// </summary>
+    PermanentlyRejected,
+
+    /// <summary>
+    /// This specific payload was accepted into the mempool at the last execution.
+    /// </summary>
+    TransientlyAccepted,
+
+    /// <summary>
+    /// This specific payload was rejected at the last execution.
+    /// </summary>
+    TransientlyRejected,
+
+    /// <summary>
+    /// We don't know anything about whether the intent could be committed or rejected.
+    /// </summary>
+    Unknown,
```

Comment:
Similarly to intent counterpart, please move it as a very first element of the enum.